DSCResources/MSFT_xDnsServerADZone/MSFT_xDnsServerADZone.schema.mof

```diff
@@ -8,4 +8,5 @@ class MSFT_xDnsServerADZone : OMI_BaseResource
     [Write, Description("DNS Server name")] String ComputerName;
     [Write, Description("Credential used to set zone"), EmbeddedInstance("MSFT_Credential")] String Credential;
     [Write, Description("Whether the DNS zone should be available or removed"), ValueMap{"Present","Absent"}, Values{"Present","Absent"}] String Ensure;
+    [Read] String CimSession;
```

Comment:
Not sure what test you want me to add. Unit-test?
Since it's a read property, it affects only the Get-TargetResource. The current module resource returns a hash-table with the 'CimSession' element in it. But since this element is missing in the schema.mof. The 'Get-DSCResource' command crashes because of the inconsistency. Other alternative would be to remove 'CimSession' from the 'xDnsServerADZone.psm1' file.

`$targetResource = @{`
`        Name = $dnsServerZone.ZoneName`
`        DynamicUpdate = $dnsServerZone.DynamicUpdate`
`        ReplicationScope = $dnsServerZone.ReplicationScope`
`        DirectoryPartitionName = $dnsServerZone.DirectoryPartitionName`
`        Ensure = if ($dnsServerZone -eq $null) { 'Absent' } else { 'Present' }`
`        CimSession = $cimSession`  **_<--- Remove this line_**
`    }`


Either way, I'm not sure what test you want me to add.